frontend/src/pages/Profile/Profile.tsx

```diff
@@ -1,8 +1,70 @@
-import { Typography } from '@mui/material'
+import {useEffect, useState} from 'react';
+import Tab from '@mui/material/Tab'
+import {Tabs} from '@mui/material'
+
+//https://stackoverflow.com/questions/66012476/how-to-show-hide-mui-tabs-based-on-a-condition-and-maintain-the-right-tab-index
 
 const Profile = () => {
+
+  const [userData, setUserData] = useState({
+    email: "",
+    firstName: "",
+    lastName: "",
+    position: "",
+    isManager: "",
+  });
+
+
+  useEffect(() => {
+    async function getData() {
+      const auth = localStorage.getItem('authorization');
+
+      if (auth) {
+        const myHeaders = new Headers();
+        myHeaders.append('authorization', auth);
+
+        const response = await fetch("/employee/profile", {
```

Comment:
idrk what axios is, but it's what the rest of the project is using so we should probably stay consistent. You can copy paste it from somewhere else, it mostly looks the same as what's here.

Reply:
the pure usage of axios is not working here because axios cannot deal with the proxy thing. However, useAxios (axios-react-hook) works with the proxy. You can check the codebase for the jobFeed page:

  const [{ data }] = useAxios({
    url: 'http://localhost:5000/jobs',
    headers: { 
      'Authorization': localStorage.getItem('authorization')
    },
    params: { 
      searchString,
      tags,
      minSalary,
      maxSalary,
      minExperience,
      maxExperience
    },
  })